inventory/host_vars/www.openfoodnetwork.org.uk/config.yml

```diff
@@ -8,6 +8,10 @@ unicorn_timeout: 120
 
 swapfile_size: 1G
 
+certbot_domains:
+  - www.openfoodnetwork.org.uk
+  - openfoodnetwork.org.uk
+
```

Comment:
I don't understand the need of a second certificate. Wouldn't a redirect at nginx level do the job?

Reply:
It adds both domains to a single certificate. It's more useful for subdomains.

Reply:
I don't like the `www` in the URL basically :joy: and I would not add a subdomain that would not be used to a certificate. But that's just me maybe. Don't bother, move on :muscle:

Reply:
Mmm that's actually the standard way to deal with this common issue isn't it @enricostano? What do experts do?🤣

Reply:
Confirmed Katuma had the same issue as UK:

![FAIL](https://user-images.githubusercontent.com/9029026/58899082-f8bfb900-86f3-11e9-84d5-8c50f135d385.png)

and the issue is now fixed. :tada: